pyface/ui/qt4/util/event_loop_helper.py

```diff
@@ -52,6 +52,10 @@ def event_loop_with_timeout(self, repeat=2, timeout=10.0):
         timeout: float, optional, keyword only
             Number of seconds to run the event loop in the case that the trait
             change does not occur. Default value is 10.0.
+
+        Notes
+        -----
+            `timeout`  is rounded to an integer.
```

Comment:
This isn't quite right: it suggest that the timeout is being rounded to an integer number of seconds. How about something like: "`timeout` is rounded to an integer number of milliseconds`?

Reply:
Actually, that's a bit clumsy. Maybe better: "The timeout is rounded to the nearest millisecond"

Reply:
Thanks for catching that. Fixed in a928735